pkg/provider/azure_loadbalancer.go

```diff
@@ -42,12 +41,14 @@ import (
 	servicehelpers "k8s.io/cloud-provider/service/helpers"
 	"k8s.io/klog/v2"
 	"k8s.io/utils/pointer"
+	"k8s.io/utils/ptr"
```

Comment:
Can we use k8s.io/utils/pointer?

Reply:
It looks like the `k8s.io/utils/pointer` is just a wrapper for `k8s.io/utils/ptr`. Plus, the package had been deprecated and we should use ptr instead: https://github.com/kubernetes/utils/blob/master/pointer/pointer.go#L17-L19.